v1.6.0 release notes fixes

xaur opened the pull request.

xaur committed:
dcrwallet: fix unwanted paragraph breaks

xaur committed:
decrediton: fix paragraph indents

xaur committed:
dcrd: restore dropped chunk of dcrd relnotes

xaur commented:
missing dcrd chunk restored from https://github.com/decred/dcrd/blob/master/docs/release-notes/release-notes-1.6.0.md

xaur committed:
decrediton: add subheadings for killer features

And move the peer count bullet to Other where it looks better.

xaur committed:
decrediton: left-hand sidebar

Reads better than left-side sidebar.

xaur committed:
decrediton: reword menu sentence

xaur committed:
decrediton: reword mixing restrictions for clarity

xaur commented:
Using small commits for easier review.

xaur committed:
decrediton: more vspd goodness and recommend to migrate

xaur committed:
decrediton: vspd no redeem scripts, no address reuse

matthawkins90 commented on release-notes.md:
```diff
@@ -1254,11 +1254,11 @@ follow what dcrwallet is doing, there is a log window below.  In the future,
-account.  Additionally, we have restricted address generation to the mixed 
-account, since only funds that have been properly mixed should be allowed to 
-end up there.
+Once privacy is enabled spending to external addresses is only allowed from the 
```
Consider adding a comma for clarity: "Once privacy is enabled, spending to..."

xaur replied:
Added.

matthawkins90 commented:
Otherwise, good updates!

xaur committed:
decrediton: privacy: add comma for clarity

xaur committed:
dcrwallet: hard fork to consensus change

xaur committed:
dcrwallet: group new features by topic

xaur committed:
decrediton: release cannot be proud

xaur committed:
decrediton: remove specifics from the intro

xaur committed:
decrediton: reorder, dev stuff to end

xaur committed:
dcrlnd: remove excess indentation

xaur committed:
dcrlnd: fix wrapping

xaur committed:
dcrlnd: fix heading levels

xaur committed:
dcrlnd: add vulns section from dcrlnd relnotes

xaur committed:
dcrdex: add explicit markdown links

GitHub commit links are not automatically converted to nice short links
because they point to a different repository.

xaur committed:
dcrdex: fix commit link

xaur committed:
dcrd: drop colon from headings

Trailing colon is not used anywhere else and even not in dcrd's other
headings.

xaur committed:
use h1 for root release heading

It doesn't make sense that release heading is h2 while its members
are h1.

xaur committed:
all: use Title Case consistently

I'm ok with either Title Case or Sentence case, but not the mix.

matthawkins90 commented on release-notes.md:
```diff
@@ -1263,53 +1457,53 @@ Please read the [initial release (v0.1.0) notes](https://github.com/decred/dcrde
+the account ID. ([1a5c070](https://github.com/decred/dcrdex/commit/1a5c070196ab7899214b524a9c681168fbdcfd75))
+- Focus password field in order dialogs. ([5eb9fb2](https://github.com/decred/dcrdex/commit/5eb9fb2de51722158eaf1d2122c11f30154bd9b3))
+- Colorize the "Side" column in the orders table. ([83b07cd](https://github.com/decred/dcrdex/commit/83b07cd08a8a7ecced012335092c0f196d7fcfb0))
 - The registration fee address is no longer logged if their is a funding error 
```
(Grammar) *there

xaur replied:
Nice catch, fixed.

matthawkins90 commented on release-notes.md:
```diff
@@ -1263,53 +1457,53 @@ Please read the [initial release (v0.1.0) notes](https://github.com/decred/dcrde
+- Focus password field in order dialogs. ([5eb9fb2](https://github.com/decred/dcrdex/commit/5eb9fb2de51722158eaf1d2122c11f30154bd9b3))
+- Colorize the "Side" column in the orders table. ([83b07cd](https://github.com/decred/dcrdex/commit/83b07cd08a8a7ecced012335092c0f196d7fcfb0))
 - The registration fee address is no longer logged if their is a funding error 
 since there is nothing a user can do with the address other than shoot their 
```
(Grammar) *themselves in the foot

xaur replied:
Haha. Fixed.

xaur committed:
dcrwallet: add code contributors

xaur committed:
dcrdex: typos

xaur committed:
decrediton: standard code contributors section

xaur committed:
dcrwallet: less heavy contributors title

xaur commented:
@alexlyp @jrick I added Code Contributors for dcrwallet and decrediton to match other projects but this can be reverted if it's not something you wanted.
